x-pack/solutions/observability/plugins/streams/server/lib/streams/helpers/retry.ts

```diff
@@ -48,11 +47,6 @@ export const retryTransientEsErrors = async <T>(
       await setTimeout(retryDelaySec * 1000);
       return retryTransientEsErrors(esCall, { logger, attempt: retryCount });
     }
-
```

Comment:
Why do we not need this anymore?

Reply:
It doesn't look like it's needed, it was only used in the route handler to convert these into 403 status codes - but we do that now generally for any ES error

Reply:
Ah, so this is doing the same thing, right? https://github.com/elastic/kibana/pull/207858/files#diff-f47a2a1be7dccc82ad1df7269048f31d722a7afc33e2fb7b5e7ea0b989d6bd5fR30

Can be resolved if yes

Reply:
yep